cmake/external.cmake

```
@@ -141,3 +141,34 @@ function(link_x11 target_name)
         target_link_libraries(${target_name} PRIVATE ${X11_Xtst_LIB})
     endif()
 endfunction()
+
+# libvpx
+function(link_vpx target_name)
+    if (TG_OWT_VPX_PACKAGED_BUILD)
+        find_package(PkgConfig REQUIRED)
+        pkg_check_modules(VPX vpx)
+        target_include_directories(${target_name} PRIVATE ${VPX_INCLUDE_DIRS})
+        target_link_libraries(${target_name} PRIVATE ${VPX_LINK_LIBRARIES})
+    else()
```

Probably, it should be effectively like this
```suggestion
    endif()
    if (NOT VPX_FOUND)
```